app/services/court_postcode_checker.rb

```diff
@@ -0,0 +1,33 @@
+require 'courtfinder_api'
+
+class CourtPostcodeChecker
```

Comment:
Can you put this service inside the namespace C100App? That way, mutant will get to test it automatically (anything inside the C100App namespace gets mutant-tested).

Reply:
done